chartfx-math/src/main/java/de/gsi/math/MathGenBase.java

```diff
@@ -4531,31 +3346,30 @@ public static double Student(double T, double ndf) {
      * @param ndf number of degrees of freedom
      * @return cumulative distribution function of Student's t-distribution
      */
-    public static double StudentI(double T, double ndf) {
+    public static double studentI(double T, double ndf) {
         final double r = ndf;
-        final double si = (T > 0) ? (1 - 0.5 * BetaIncomplete((r / (r + T * T)), r * 0.5, 0.5))
-                                  : 0.5 * BetaIncomplete((r / (r + T * T)), r * 0.5, 0.5);
+        final double si = (T > 0) ? (1 - 0.5 * betaIncomplete((r / (r + T * T)), r * 0.5, 0.5))
+                                  : 0.5 * betaIncomplete((r / (r + T * T)), r * 0.5, 0.5);
         return si;
     }
 
-    // TODO: continue here
-    public static double StudentQuantile(double p, double ndf, boolean lower_tail) {
+    /**
+     * Computes quantiles of the Student's t-distribution
+     *  When the 3rd argument lower_tail is true (default)-  the algorithm returns such x0, that  P(x < x0)=p
+     *  upper tail (lower_tail is false)- the algorithm returns such x0, that  P(x > x0)=p
+     *
+     * the algorithm was taken from
+     * G.W.Hill, "Algorithm 396, Student's t-quantiles"
+     * "Communications of the ACM", 13(10), October 1970
+     * @param p the probability, at which the quantile is computed
+     * @param ndf the number of degrees of freedom of the
+     * @param lower_tail
+     */
+    public static double studentQuantile(double p, double ndf, boolean lower_tail) {
```

Comment:
Codacy found an issue: [The method 'studentQuantile(double, double, boolean)' has an NPath complexity of 392, current threshold is 200](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)